docs/src/topics/troubleshooting.md

```diff
@@ -38,9 +38,28 @@ Node are not ready because they need cni to be ready.
 If your vm is never in running phase and but still in provisonned phase, please look at the cloud init log of your vm.
 
 
+### Trouble with e2etest path
+You should clean a previous installation before launching e2etest:
+```
+make uninstall-clusterapi
+kubectl get crd -A | grep x-k8s.
+```
+
+You should delete all cluster-api's CRD.
+If deletion is blocked, patch them.
```

Comment:
If deletion is blocked, you can use Finalizers to delete crds by patching them one by one

Reply:
Done